ci/run_benchmarks.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+# Support invoking run_benchmarks.sh outside the script directory
+cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../dask_cuda
+
+rapids-logger "Run local benchmark"
```

Comment:
```suggestion
echo "Run local benchmark"
```

By convention, we try to keep any stuff from `gha-tools` like `rapids-logger` out of these `ci/run_*.sh` scripts, as those scripts are intended to be run outside of CI as well (like in DLFW builds).

Could we use `echo` here instead?

Reply:
Makes sense, however, I prefer `rapids-logger` as it makes it easier to visually identify that in the logs, so I moved the calls to the `test_*.sh` files instead.